src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -4395,6 +4395,149 @@ public static void DerivedTypeWithDifferentOverrides()
     }
 #endif
 
+    [Fact]
+    public static void Xml_DefaultNamespaceChange_SimpleArray_ObjectAsRoot()
+    {
+        SimpleType[] x = new SimpleType[] { new SimpleType { P1 = "abc", P2 = 11 }, new SimpleType { P1 = "def", P2 = 12 } };
+#if uap
```

Comment:
I have changed all `uap` to `uapaot` and run the test target `uapaot` , this case failed targeting `uapaot `in the ReflectionOnly test project. So I use `#if uapaot && !ReflectionOnly` so that the case can pass.